.github/workflows/buildpack/upgrade.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+set -e
+
+check_parameters() {
+  echo "-- check_parameters"
+
+  if [ -z "${IMAGE_NAME}" ] || [ -z "${IMAGE_VARIANT}" ] || [ -z "${GITHUB_TOKEN}" ] || [ -z "${GITHUB_ACTOR}" ] || [ -z "${GITHUB_DOCKER_REPOSITORY}" ]; then
+    echo "\${IMAGE_NAME}, \${IMAGE_VARIANT}, \${GITHUB_TOKEN}, \${GITHUB_ACTOR} and \${GITHUB_DOCKER_REPOSITORY} need to be defined."
```

Comment:
We don't need ``GITHUB_TOKEN`` and ``GITHUB_ACTOR`` anymore, do we?
And we can rename ``GITHUB_DOCKER_REPOSITORY`` to ``DOCKER_REPOSITORY``.
That way it suddenly becomes a generic script that can be useful wherever we may run it and whereever we may want to push the images to and deserves its place in ``scripts/ci/`` :-).

Reply:
To be honest, I'd still also hard-code ``IMAGE_NAME``, reducing the number of parameters to 2, which I'd then pass as arguments instead of via environment variables, giving up on the step idea ;-). But I'm not dead-set on that...

Reply:
I would like to have `IMAGE_NAME` like it is, because maybe we can later use the same script for other docker images.